docs/sources/panels/configure-panel-options/index.md

```diff
@@ -0,0 +1,140 @@
+---
+aliases:
+  - /docs/grafana/latest/panels/add-panels-dynamically/about-repeating-panels-rows/
+  - /docs/grafana/latest/panels/add-panels-dynamically/configure-repeating-rows/
+  - /docs/grafana/latest/panels/add-panels-dynamically/configure-repeating-panels/
+  - /docs/grafana/latest/panels/add-panels-dynamically/
+  - /docs/grafana/latest/panels/repeat-panels-or-rows/
+  - /docs/grafana/latest/panels/working-with-panels/add-title-and-description/
+  - /docs/grafana/latest/panels/working-with-panels/view-json-model/
+  - /docs/grafana/latest/panels/configure-panel-options/
+title: Configure panel options
+menuTitle: Configure panel options
+weight: 150
+keywords:
+  - panel
+  - dynamic
+  - rows
+  - add
+  - title
+  - description
+  - JSON model
+---
+
+# Configure panel options
+
+A Grafana panel is the user interface you use to define a data source query, and transform and format data that appears in visualizations.
+
+A panel editor includes a query builder and a series of options that you can use to transform data and add information to your panels.
+
+This topic describes how to:
+
+- open a panel for editing
+- add a panel title and description
+- view a panel JSON model
+- add repeating rows and panels
+
+## Edit a panel
+
+After you add a panel to a dashboard, you can open it at any time to change change or update queries, add data transformation, and change visualization settings.
+
+1. Open the dashboard that contains the panel you want to edit.
+
+1. Click in the panel title and select **Edit**.
+
+   To use a keyboard shortcut to open the panel, hover over the panel and press `e`.
+
+   The panel opens in edit mode.
```

Comment:
Nitpick, these empty lines between these sentences look a bit strange not sure if they are intentional or not :shrug:  (not very important, just found it curious) 
![Screenshot from 2022-08-10 11-45-03](https://user-images.githubusercontent.com/10999/183870846-f4368ef6-0ebe-480e-83c5-20f41f2a661e.png)

Reply:
Yeah, I agree. I can fix that. Thanks.

Reply:
Done.